src/utils.ts

```diff
@@ -7,3 +10,22 @@ export function parseNumber(str?: string): number | undefined {
 export function _throw(msg: string): never {
     throw msg;
 }
+
+export async function getCoordsFromAddress(address: string): Promise<any> {
```

Comment:
We need to write a return type for this function. using `any` is a bad idea.  For now you could probably just add an interface `ICoords` that lives next to this function.

Reply:
mosdef!